src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -57,6 +57,18 @@
       + "the connection string as hostname1:port1,hostname2:port2,hostname3:port3/chroot/path.";
   public static final String ZOOKEEPER_CONNECT_DEFAULT = "localhost:2181";
 
+  public static final String BOOTSTRAP_SERVERS_CONFIG = "bootstrap.servers";
+  private static final String
+      BOOTSTRAP_SERVERS_DOC =
```

Comment:
Added PLAINTEXT://localhost:9092 as a default value. 
https://github.com/confluentinc/kafka-rest/pull/216/commits/b8a2cf58273d8038ffe30a630f52d1490b163002
Thank you.